src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/prison/register/PrisonDto.kt

```diff
@@ -6,9 +6,6 @@ import io.swagger.v3.oas.annotations.media.Schema
 data class PrisonDto(
```

Comment:
I would call it PrisonNamesDto

Reply:
Updated Dto name and some accompanying logic